File: app/src/main/java/com/djrausch/billtracker/receivers/BootReceiver.java
package com.djrausch.billtracker.receivers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import com.djrausch.billtracker.util.AlarmUtil;

/**
 * Created by white on 7/26/2016.
 */
public class BootReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if (intent.getAction().equals("android.intent.action.BOOT_COMPLETED") || intent.getAction().equals("android.intent.action.MY_PACKAGE_REPLACED")) {
            AlarmUtil.setDailyAlarm(context);
        }
    }
}

File: app/src/main/java/com/djrausch/billtracker/receivers/BootReceiver.kt
package com.djrausch.billtracker.receivers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

import com.djrausch.billtracker.util.AlarmUtil

/**
 * Created by white on 5/29/2016.
 */
class BootReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action == "android.intent.action.BOOT_COMPLETED" || intent.action == "android.intent.action.MY_PACKAGE_REPLACED") {
            AlarmUtil.setDailyAlarm(context)
        }
    }
}
